plugins/modules/packaging/language/gem.py

```diff
@@ -62,6 +62,17 @@
       These gems will be independent from the global installed ones.
       Specifying this requires user_install to be false.
     required: false
+  bindir:
+    type: path
+    description:
+    - Install executables into a specific directory.
+    version_added: 3.3.0
+  norc:
+    type: bool
+    default: false
+    description:
+    - Avoid loading any C(.gemrc) file. Ignored for RubyGems prior to 2.5.2.
```

Comment:
```suggestion
    - Avoid loading any C(.gemrc) file. Ignored for RubyGems prior to 2.5.2.
    - From community.general 4.0.0 on, there will be no default value for this option.
    - From community.general 5.0.0 on, default will be changed to `true`.
```

Reply:
Ah, one a second thought! No default option for a boolean means what, it puts `norc` or `norc` at last? :)
Better to reword this, maybe remove the line `from c.g. 4` from docs and just show deprecation warning to users using this module, not specifying explicit value for `norc`.
What do you think about this?

Reply:
Makes sense. Where can I see the correct format for deprecation warning?

Reply:
Well, it's not now anyway. How can we make sure that we don't forget to add deprecation warning later?

Reply:
Good question, I don't know.

Maybe we should have a label for these kinds of PR, so ansibullbot can mention author when the time comes (4.0.0 has been released)?

Reply:
If something is for the next major release after the one that's currently already out, it's pretty simple: create a follow-up PR. That one can be merged, but not backported - then it won't show up in a 3.x.0 release, but only in 4.0.0. :)

(Basically `main` is what will become `stable-4` once we're preparing 4.0.0. For some PRs that have already been merged that will only appear in 4.0.0, see https://github.com/ansible-collections/community.general/issues?q=label%3Abreaking_change+is%3Aclosed - all PRs there that haven't included in 3.0.0 are waiting for 4.0.0.)